src/DripsHub.sol

```diff
@@ -427,5 +434,23 @@ abstract contract DripsHub {
     /// @param assetId The used asset ID
     /// @param amt The transferred amount.
     /// Positive to transfer funds to the user, negative to transfer from them.
-    function _transfer(uint256 assetId, int128 amt) internal virtual;
+    function _transfer(uint256 assetId, int128 amt) internal {
+        IERC20 erc20 = IERC20(address(uint160(assetId)));
+        if (amt > 0) {
+            reserve.withdraw(erc20, msg.sender, uint128(amt));
+        } else if (amt < 0) {
+            reserve.deposit(erc20, msg.sender, uint128(-amt));
```

Comment:
I would still prefer to make it vice-versa to be aligned with the `balanceDelta`.

```solidity
/// Positive to add funds to the drips balance, negative to remove them.
```

Reply:
Agree, fixed